contracts/cw20-dao-crowdfund/src/state.rs

```diff
@@ -33,13 +36,20 @@ pub enum Status {
         /// The token price in number of tokens per native token
         /// (ex. uJuno).
         token_price: Decimal,
+        initial_gov_token_supply: Uint128,
     },
     /// The DAO has closed the campaign. Refunds are avaliable but no
     /// new funds may be added.
-    Cancelled { token_price: Decimal },
+    Cancelled {
+        token_price: Decimal,
+        initial_gov_token_supply: Uint128,
+    },
     /// The campaign has met its funding goal. Tokens may now be
     /// exchanged for governance tokens in the DAO.
-    Funded { token_price: Decimal },
+    Funded {
+        token_price: Decimal,
+        initial_gov_token_supply: Uint128,
+    },
```

Comment:
Can we change these to `initial_gov_token_balance` since it is the balance of the campaign and not actually the supply?

Reply:
Done. Sounds good.